en/sensor/vectornav.md

```diff
@@ -0,0 +1,119 @@
+# VectorNav
+
+VectorNav produces [IMU/AHRS](https://www.vectornav.com/resources/inertial-navigation-primer/theory-of-operation/theory-ahrs), [GNSS/INS](https://www.vectornav.com/resources/inertial-navigation-primer/theory-of-operation/theory-gpsins), [Dual GNSS/INS](https://www.vectornav.com/resources/inertial-navigation-primer/theory-of-operation/theory-gnsscompass) systems.
+
+PX4 can use these as an [external INS](../sensor/inertial_navigation_systems.md), bypassing/replacing the EKF2 estimator, or as a source of raw sensor data provided to the estimator.
+
+![VN-300](../../assets/hardware/sensors/inertial/vn-300-smd-rugged.png)
+
+PX4 driver supports [all VectorNav sensors](https://www.vectornav.com/store/products).
+In particular the following systems are recommended:
+
+- **VN-200 GNSS/INS:** Recommended for fixed-wing systems without hovering, where static heading is not necessary.
+- **VN-300 DUAL GNSS/INS:** Recommended for multicopter systems where hovering and low dynamics requires the use of static heading.

```

Comment:
These are GNSS too. Is there any reason not to add this to the table here: https://docs.px4.io/main/en/gps_compass/#supported-gnss ?

Reply:
I agree they're technically supported GNSS formats, but they only work with the VN driver enabled so I don't know that they would qualify for documenting in this case

Reply:
IMO they "qualify" if using the VectorNav you no longer need to purchase a GPS. However we'd need to make clear in the docs that these require different setup. Make sense?